tests/getinfo_test.py

```diff
@@ -30,6 +30,7 @@ def test_getinfo(self):
         assert self.curl.getinfo(pycurl.SPEED_DOWNLOAD) > 0
         self.assertEqual(7, self.curl.getinfo(pycurl.SIZE_DOWNLOAD))
         self.assertEqual('http://%s:8380/success' % localhost, self.curl.getinfo(pycurl.EFFECTIVE_URL))
+        self.assertEqual("GET", self.curl.getinfo(pycurl.EFFECTIVE_METHOD))
```

Comment:
Can we guard this one check for libcurl > 7.72.0?

Reply:
I've added a separate test for it guarded by a version check.

Reply:
I was thinking you could just guard that one assert with an `if not pycurl_version_less_than()` but this works too.